src/imageio/format/jpeg.c

```diff
@@ -396,6 +394,10 @@ int write_image(dt_imageio_module_data_t *jpg_tmp, const char *filename, const v
   jpeg_finish_compress(&(jpg->cinfo));
   jpeg_destroy_compress(&(jpg->cinfo));
   fclose(f);
+
+  if(exif && exif_len > 0 && exif_len < 65534)
```

Comment:
@mazhe
Why does it check length?
Both other dt_exif_write_blob() invocations don't

exiv2 does throw if there are any issues, so i'm not sure what harm it could cause